source/options.css

```diff
@@ -8,36 +10,31 @@ html.is-edgium {
 	min-width: auto;
 }
 
-:root {
-	--github-green: #28a745;
-	--github-red: #cb2431;
-}
-
 body {
 	font-family: system-ui, sans-serif;
 	font-size: 14px;
 	margin: 0;
 	padding: 16px;
```

Comment:
I think you can remove many of the properties here. Maybe also in other selectors. Try to remove as much custom CSS as possible.

Reply:
For some reason Firefox had a grey background in dark mode, so I left these out. Now that's gone, I'll remove this.